.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -5,14 +5,20 @@ about: Create a report to help us squash bugs!
 ---
 
 <!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
-v                            ✰  Thanks for opening an issue! ✰    
+v               ✰  Thanks for opening an issue! ✰    
 v    Before smashing the submit button please review the template.
 v    Please also ensure that this is not a duplicate issue :)  
 ☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
 
-## Summary of Bug
+## Bug Summary
 
 <!-- Describe the issue you're encountering -->
+<!-- Apply relevant labels to indicate:
```

Comment:
I discarded this. It seems we have more to work with labels than outside contributors. In addition, more detail on this can be find in `CONTRIBUTING.md`